src/content/docs/zh-cn/guides/styling.mdx

```diff
@@ -22,38 +20,41 @@ Astro 的设计为了使设计和编写 CSS 变得轻而易举。直接在 Astro
 
 Astro `<style>` 标签内的 CSS 规则**默认自动限定范围**。作用域样式在幕后编译，只适用于写在同一组件内的 HTML。你在 Astro 组件中编写的 CSS 会自动封装在该组件中。
 
-```diff
+```astro del={2,5} ins={3,6} ins=":where(.astro-HHNQFKH6)"
 <style>
--  h1 { color: red; }
-+  h1.astro-HHNQFKH6 { color: red; }
--  .text { color: blue; }
-+  .text.astro-HHNQFKH6 { color: blue; }
+  h1 { color: red; }
+  h1:where(.astro-HHNQFKH6) { color: red; }
+
+  .text { color: blue; }
+  .text:where(.astro-HHNQFKH6) { color: blue; }
 </style>
```

Comment:
This part changed a bit, could you please update it based on these changes? 🙌 https://github.com/withastro/docs/commit/a916911d52ddd288213375a9e28e75da872998d3